### DISCUSSION TITLE
THYDU𝕏: Decentralized Application with DU𝕏 Token Integration on Optimism

### DISCUSSION THREAD

**Discussion by THYDUX** [ethereum-optimism/developers#613] [Show and tell]
### Did you check the documentation?

- [x] Yes

### Did you check for duplicate questions?

- [x] Yes

### Issue Description

Application development 

### Additional Information

DU𝕏 is your new crypto,designed for smart trading & investment. Customize your strategy,trade confidently, & watch your portfolio grow.

**Comment by sbvegan** [comment]
Thanks for sharing! I moved this over to the show and tell section. Please feel free to add more details and links that others can check out.

**Comment by SammySSSS5** [comment]
Hey nice project, I'd love to contribute to your project.
